openshift/README.md

```diff
@@ -8,32 +8,49 @@ Red Hat OpenShift is an open source container application platform based on the
 
 ### Installation
 
-To install the Agent, see the [Agent installation instructions][2] for Kubernetes. The default configuration targets OpenShift 3.7.0+ and OpenShift 4.0+, as it relies on features and endpoints introduced in this version.
+The default configuration Datadog provides targets OpenShift 3.7.0+ and OpenShift 4.0+, as it relies on features and endpoints introduced in this version.
```

Comment:
Is there a different word we can use besides "targets" here? I feel like it makes the sentence hard to read, and I'm not 100% sure what is meant by it in this sentence, either.

Reply:
In the public docs PR, we have `This core configuration supports OpenShift 3.11 and OpenShift 4, but it works best with OpenShift 4.`

https://github.com/DataDog/documentation/pull/24920/files#diff-b30bababc38481fe44f2b7728dcbec2dfa72e0ecf8885d7b0e12d89ac7d8dcb0R344

Reply:
Updated to match that same sample from the main public docs. 

> This core configuration supports OpenShift 3.11 and OpenShift 4, but it works best with OpenShift 4.
>
> To install the Agent, see the ...